packages/ui/src/Button/index.tsx

```diff
@@ -28,6 +31,7 @@ const compact = css<StyledButtonProps>`
   padding-right: ${props => props.theme.spacing(props.$iconOnly ? 2 : 4)};
   height: 32px;
   min-width: 32px;
+  width: max-content;
```

Comment:
Fixes a weird issue where the button was sometimes full-width when it shouldn't have been